File: 2017-05-02/PowerManager/app/src/main/java/br/ufpe/cin/if1001/powermanager/MainActivity.java
package br.ufpe.cin.if1001.powermanager;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        final Button button1 = (Button) findViewById(R.id.btn_keepScreen1);
        button1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                //startActivity(new Intent(MainActivity.this, SemThreads.class));
            }
        });

        final Button button2 = (Button) findViewById(R.id.btn_keepScreen2);
        button2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                //startActivity(new Intent(MainActivity.this, SemThreads.class));
            }
        });

        final Button button3 = (Button) findViewById(R.id.btn_wakelock);
        button3.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                //startActivity(new Intent(MainActivity.this, SemThreads.class));
            }
        });

        final Button button4 = (Button) findViewById(R.id.btn_wakelock2);
        button4.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                //startActivity(new Intent(MainActivity.this, SemThreads.class));
            }
        });
    }

    @Override
    protected void onResume() {
        super.onResume();

    }
}

File: 2018.1/2018-04-18/Managers/app/src/main/java/br/ufpe/cin/if710/managers/MainActivity.kt
package br.ufpe.cin.if710.managers

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import br.ufpe.cin.if710.managers.alarm.AlarmManagerActivity
import br.ufpe.cin.if710.managers.jobscheduler.JobSchedulerActivity
import br.ufpe.cin.if710.managers.location.LocationMapsActivity
import br.ufpe.cin.if710.managers.notification.NotificationManagerActivity
import br.ufpe.cin.if710.managers.phonesms.PhoneSmsActivity
import br.ufpe.cin.if710.managers.pkg.PackageManagerActivity
import br.ufpe.cin.if710.managers.power.PowerManagerActivity
import br.ufpe.cin.if710.managers.sensor.SensorManagerActivity
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btn1.setOnClickListener {
            startActivity(Intent(applicationContext, NotificationManagerActivity::class.java))
        }
        btn2.setOnClickListener {
            startActivity(Intent(applicationContext, AlarmManagerActivity::class.java))
        }
        btn3.setOnClickListener {
            startActivity(Intent(applicationContext, JobSchedulerActivity::class.java))
        }
        btn4.setOnClickListener {
            startActivity(Intent(applicationContext, PowerManagerActivity::class.java))
        }
        btn5.setOnClickListener {
            startActivity(Intent(applicationContext, SensorManagerActivity::class.java))
        }
        btn6.setOnClickListener {
            startActivity(Intent(applicationContext, PackageManagerActivity::class.java))
        }
        btn7.setOnClickListener {
            startActivity(Intent(applicationContext, PhoneSmsActivity::class.java))
        }
        btn8.setOnClickListener {
            startActivity(Intent(applicationContext, LocationMapsActivity::class.java))
        }
    }

    companion object {

        val PACKAGE_NAME = "br.ufpe.cin.if710.managers"
    }
}